tools/coq_dune.ml

```diff
@@ -0,0 +1,210 @@
+(* ocamlfind ocamlopt -linkpkg -package str coq_dune.ml -o coq_dune.opt *)
```

Comment:
Licence header is missing.

Also a line explaining what this utility is supposed to do would help.

The build instructions should be removed, they belong to the build system.

Reply:
Header added, note added, bootstrap instructions kept as they important, note added.